### PR TITLE
feat(suggest): add icons on chips

### PR DESCRIPTION
**Pull request by CalinaCristian**
Adds support for icons in chips.

image

### CODE REVIEW

File: projects/angular/components/ui-suggest/src/ui-suggest.component.html [hunk 1]
```diff
@@ -102,6 +102,14 @@
                           [matTooltip]="intl.translateLabel(option.text)"
                           (removed)="deselectItem(option)"
                           class="chip-selectable">
+                    <mat-icon *ngIf="!!option?.icon?.svgIcon && displayIconInChip"
```
Comment (anbalase) [inline projects/angular/components/ui-suggest/src/ui-suggest.component.html:105]: If consumers already provide `option.icon` why should we also add a flag for rendering?
wdyt?

Comment (anbalase) [reply projects/angular/components/ui-suggest/src/ui-suggest.component.html:105]: My first thought would be to not flag this added feature.

Comment (anbalase) [reply projects/angular/components/ui-suggest/src/ui-suggest.component.html:105]: maybe switch the flag to true by default to keep the logic but have it implicit as a recommendation. This is a cosmetic bonus and not a breaking change.

Comment (CalinaCristian) [reply projects/angular/components/ui-suggest/src/ui-suggest.component.html]: I'll remove the flag entirely 👍

**Review by anbalase — APPROVED** [review]

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,8 @@
+# v14.2.0 (2023-01-30)
+* **grid** add icon on chips
```
Comment (anbalase) [inline CHANGELOG.md:2]: nit: it's actually a suggest feat right :D

**Pull request merged**